README.md

```diff
@@ -67,9 +67,9 @@ Various static checks:
 Double matrix across two dimensions:
 
   - All three major operating systems: Linux, macOS, and Windows.
-  - Dart SDK: stable, beta, dev, or a specific version string.
+  - Dart SDK: Latest stable, beta & dev plus two specific versions.
```

Comment:
I can, if I want, setup a matrix with all dev versions we've ever published. It seems like the new text describes the specific example which happens to use two versions? The new text implies that the action is limited to two specific versions. Also, the `or` was better because you can also have only a specific version.

Reply:
The text here describes what is in the example below, not the limits of what is possible. I tried to clarify the text, PTAL

Reply:
LGTM